### PR TITLE
news: Add 10 items

### PR DESCRIPTION
**Pull request by xaur**
- Add 2 Bison Relay non-RC releases
- Add 4 Decred Journal issues
- Add 4 Coverage items
  - Selection criteria used is "anything notable, ideally involving people/orgs external to Decred"
- Fix dead decredmagazine.com links to cypherpunktimes.com
  - Not expecting the redirects will be fixed anytime soon

I still don't have a good rule to follow for choosing which `icon` to use. If anyone has one, please open an issue, tag me and I will apply it to make our news icons consistent.

### CODE REVIEW

File: src/data/news/decred_journals.yml [hunk 1]
```diff
@@ -1,75 +1,107 @@
+-
+  Title: "Decred Journal, July 2023"
+  Permalink: "https://www.cypherpunktimes.com/decred-journal-july-2023/"
+  Params:
+    author: "Decred Journal Team"
+    sortDate: "2023-07-22"
```
Comment (jholdstock) [inline src/data/news/decred_journals.yml:6]:
```suggestion
    sortDate: "2023-08-22"
```

**Review by jholdstock — CHANGES_REQUESTED** [review]

**Comment by jholdstock** [comment]
re: icons
https://github.com/decred/dcrweb/pull/1099#pullrequestreview-1194813718

File: src/data/news/coverage.yml [hunk 2]
```diff
@@ -1,3 +1,27 @@
...
+-
+  Title: "Crypto Integrations - Cake Wallet"
+  Permalink: "https://www.youtube.com/watch?v=0KKsD4ZhZn0"
+  Params:
+    author: "Phoenix Green, Tivra"
+    sortDate: "2023-08-02"
+    icon: "broadcasts.svg"
+    outlet: "Decred YouTube"
+-
+  Title: "The Future Is Now: Jake Yocom-Piatt Of Decred.org On How Their Technological Innovation Will Shake Up The Tech Scene"
```
Comment (jholdstock) [inline src/data/news/coverage.yml:18]: It's minor so feel free to ignore, but I always hate seeing caps in URLs. `decred.org` looks way more natural imo.

File: src/data/news/decred_journals.yml (hunk 1 above)
Comment (xaur) [reply src/data/news/decred_journals.yml]: Good catch! I re-checked the diff so many times but still missed this bad paste.

File: src/data/news/coverage.yml (hunk 2 above)
Comment (xaur) [reply src/data/news/coverage.yml]: The title was directly copied from the source article. I think it is not uncommon for some websites to capitalize domain names that should really be not capitalized.

Fixed that cap just for our comfort here. In the future we should streamline all titles to `Title Case` or `Sentence case` and fix all nits like this (related to bits of #1101).

**Comment by xaur** [comment]
> re: icons [#1099 (review)](https://github.com/decred/dcrweb/pull/1099#pullrequestreview-1194813718)

Thanks for the link. Now I see what confused me back then and now again. The problem starts in the original categorization in #561: the `Broadcast, streams, videos` class in "Community", and `Decred in media` and `Press releases, interviews` classes in "Media and Events" all overlap. One news item can be all of: a (live)stream, a video, something "in the media", and an interview. Then the informal "`broadcasts.svg` for video interviews and `interview.svg` for audio only" is also confusing to me. Most other icons/categories do not have this problem.

For this PR I changed "Quantum and Cryptocurrency" to `interview.svg` because it's kind of an interview. Keeping "The Future Is Now" as `dcrInMedia.svg` because even though it's an *interview*, the microphone icon of `interview.svg` implies audio. Keeping the remaining two as `broadcasts.svg` because "broadcast" sounds like "live stream".

In the future I hope we'll define better semantics for these icons, rename them, and maybe change 1-2 icons to avoid confusion and time loss.

**Comment by xaur** [comment]
Please remind me should I squash or do you guys squash when merging?

**Comment by jholdstock** [comment]
In simple cases like this its easy to squash with GitHub. A repo maintainer may ask you to squash manually in more complicated cases so it can be done more carefully - eg. if a series of commits depend on eachother, or if their messages contain important context which may need to be updated.

**Review by jholdstock — APPROVED** [review]

**Pull request merged**